packages/image_picker/image_picker/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
As I said before, and as it says in the document I referenced above, this PR needs a version change.

Reply:
done.